### PR TITLE
7839 additional fields

### PR DESCRIPTION
**Pull request by rosiehsmith**
* Update validation after case is served

### CODE REVIEW

File: shared/src/business/entities/contacts/ContactFactory.js [hunk 1]
```diff
@@ -192,12 +193,15 @@ ContactFactory.getErrorToMessageMap = ({
  * used for getting the contact constructors depending on the party type and contact type
  *
  * @param {object} options the options object
- * @param {string} options.countryType typically either 'domestic' or 'international'
- * @param {boolean} options.isPaper is paper case
  * @param {string} options.partyType see the PARTY_TYPES map for a list of all valid partyTypes
+ * @param {string} options.status the case status
```
Comment (rachelschneiderman) [inline shared/src/business/entities/contacts/ContactFactory.js:197]: Nit: docs missing `contactInfo`

**Review by mrinsin — APPROVED** [review]

File: shared/src/business/useCases/validatePetitionerInformationFormInteractor.js [hunk 2]
```diff
@@ -14,11 +14,13 @@ exports.validatePetitionerInformationFormInteractor = ({
   contactPrimary,
   contactSecondary,
   partyType,
+  status,
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/validatePetitionerInformationFormInteractor.js:17]: Nit: docs

File: shared/src/business/useCases/validatePetitionerInteractor.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ exports.validatePetitionerInteractor = ({
   contactInfo,
   partyType,
   petitioners,
+  status,
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/validatePetitionerInteractor.js:18]: Nit: docs

File: shared/src/business/useCases/validatePrimaryContactInteractor.js [hunk 4]
```diff
@@ -12,10 +12,12 @@ exports.validatePrimaryContactInteractor = ({
   applicationContext,
   contactInfo,
   partyType,
+  status,
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/validatePrimaryContactInteractor.js:15]: Nit: docs

File: shared/src/business/useCases/validateSecondaryContactInteractor.js [hunk 5]
```diff
@@ -12,10 +12,12 @@ exports.validateSecondaryContactInteractor = ({
   applicationContext,
   contactInfo,
   partyType,
+  status,
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/validateSecondaryContactInteractor.js:15]: Nit: docs

**Review by rachelschneiderman — APPROVED** [review]

**Pull request merged**